config/chadrc.lua

```diff
@@ -42,8 +42,8 @@ local M = {}
 local highlights = require "custom.highlights"
 
 M.ui = {
-  theme_toggle = { "onedark", "everforest_light" },
-  theme = "onedark",
+  theme_toggle = { "palenight", "one_light" },
```

Comment:
@soulfresh what's your favorite theme? Maybe we add our favorites to the toggle array. These are my two. Check out palenight, it's pretty great. But whichever you want as the default is fine.

Reply:
@soulfresh Good call. Have a look now. I created an example file we can simply copy. That work? Would it make sense to move a bunch of the settings inside chadrc.lua into init.lua?